lib/factory/SelectEntryFactory.js

```diff
@@ -51,28 +52,35 @@ var selectbox = function(options, defaultParameters) {
 
 
   resource.html =
-    '<label for="camunda-' + escapeHTML(resource.id) + '"' +
+    domify('<label for="camunda-' + escapeHTML(resource.id) + '"' +
```

Comment:
I wonder why we don't do this first hand?

Reply:
Once you go DOM you cannot go back. Since we need to add _More_/_Less_ logic to descriptions they must be DOM. Therefore all parent elements must be DOM as well.